sdk/personalizer/azure-ai-personalizer/azure/ai/personalizer/aio/operations/_operations.py

```diff
@@ -0,0 +1,2922 @@
+from urllib.parse import parse_qs, urljoin, urlparse
+
+from azure.core.async_paging import AsyncItemPaged, AsyncList
+from azure.core.exceptions import (
+    ClientAuthenticationError,
+    HttpResponseError,
+    ResourceExistsError,
+    ResourceNotFoundError,
+    ResourceNotModifiedError,
+    map_error,
+)
+from azure.core.pipeline import PipelineResponse
+from azure.core.pipeline.transport import AsyncHttpResponse
+from azure.core.rest import HttpRequest
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.tracing.decorator_async import distributed_trace_async
+from azure.core.utils import case_insensitive_dict
+
+from ...operations._operations import (
+    build_evaluations_create_request,
+    build_evaluations_delete_request,
+    build_evaluations_get_request,
+    build_evaluations_list_request,
+    build_events_activate_request,
+    build_events_reward_request,
+    build_feature_importances_create_request,
+    build_feature_importances_delete_request,
+    build_feature_importances_get_request,
+    build_feature_importances_list_request,
+    build_log_delete_request,
+    build_log_get_properties_request,
+    build_model_get_properties_request,
+    build_model_get_request,
+    build_model_import_method_request,
+    build_model_reset_request,
+    build_multi_slot_events_activate_request,
+    build_multi_slot_events_reward_request,
+    build_multi_slot_rank_request,
+    build_policy_get_request,
+    build_policy_reset_request,
+    build_policy_update_request,
+    build_rank_request,
+    build_service_configuration_apply_from_evaluation_request,
+    build_service_configuration_get_request,
+    build_service_configuration_update_request,
+)
+from .._vendor import MixinABC
+
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
+JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
+T = TypeVar("T")
+ClsType = Optional[Callable[[PipelineResponse[HttpRequest, AsyncHttpResponse], T, Dict[str, Any]], Any]]
+
+
+class ServiceConfigurationOperations:
+    """
+    .. warning::
+        **DO NOT** instantiate this class directly.
+
+        Instead, you should access the following operations through
+        :class:`~azure.ai.personalizer.aio.PersonalizerClient`'s
+        :attr:`service_configuration` attribute.
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        input_args = list(args)
+        self._client = input_args.pop(0) if input_args else kwargs.pop("client")
+        self._config = input_args.pop(0) if input_args else kwargs.pop("config")
+        self._serialize = input_args.pop(0) if input_args else kwargs.pop("serializer")
+        self._deserialize = input_args.pop(0) if input_args else kwargs.pop("deserializer")
+
+    @distributed_trace_async
+    async def get(self, **kwargs: Any) -> JSON:
```

Comment:
We should try to align the naming of methods across languages. You can do this by overriding the operationId in the swagger readme. An example here: https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cognitivelanguage/azure-ai-language-conversations/swagger/README.md?plain=1#L157

Also while looking at the Java/.NET library, it seems we've divided operations into two clients. I'm wondering if we want to do the same for Python?

Reply:
Seems Java/.NET did more of an HLC (high level client). JS did a Rest client and so I think it's okay if we stick with what's generated by autorest/DPG for Python.